production/scrapers/south_dakota_population.R

```diff
@@ -13,8 +13,18 @@ south_dakota_population_check_date <- function(x, date = Sys.Date()){
 }
 
 south_dakota_population_pull <- function(x){
-    get_src_by_attr(x, "a", attr = "href", 
-                    attr_regex = "(?i)documents/AdultPopulation.*.pdf")
+    x <- 'https://doc.sd.gov/about/stats/adult/' %>%
```

Comment:
Since you're in here, can you rename `x` to be more descriptive? Are we pulling the pdf from something that could be described as raw_html? Something else?

Reply:
I could be missing something, but I think this whole function can be much simplified by replicating the code in the date check function: 

```
 pdf_link <-   get_src_by_attr(x, "a", attr = "href", 
                    attr_regex = "(?i)documents/AdultPopulation.*.pdf")  %>%
        first()
return(pdf_link)
```